common-test/src/test/java/com/tozny/e3db/ClientTest.java

```diff
@@ -647,7 +650,11 @@ public void handle(Result<Config> r) {
         if(r.isError())
           throw new Error(r.asError().other());
 
-        clientRef.set(new ClientBuilder().fromConfig(r.asValue()).build());
+        try {
+          clientRef.set(new ClientBuilder().fromConfig(r.asValue()).build());
+        } catch (E3DBCryptoException e) {
+          throw new RuntimeException();
```

Comment:
Why is this cast to an empty RuntimeException here? - are we not adding any messaging for it?

Reply:
This is just in a test. Trying to break this out required weird unwanted signature changes so I just left it. Nothing in user facing code is unmessaged.